docs/01-overview/main-areas/application-connectivity/ac-05-useful-links.md

```diff
@@ -7,7 +7,6 @@ If you're interested in learning more about the Application Connectivity area, f
 - Perform some simple and more advanced tasks:
 
   - [Pass the access token in the request header](../../../04-operation-guides/operations/ac-01-pass-access-token-in-request-header.md)
-  - [Register an API in Application Registry](../../../04-operation-guides/operations/ac-02-api-registration.md)
   - [Provide a custom AC certificate and key](../../../04-operation-guides/operations/ac-03-application-connector-certificates.md)
```

Comment:
We can drop this one as well. I think the following ones will also be invalidated:
- Get the client certificate
- Renew a client certificate
- Revoke a client certificate
- Rotate the Root certificate ...

We need to revisit those tutorials:
- Register a service
- Register a secured API

Those show how to execute calls to Application Registry, we need to shift those to modify Application CR directly - this will be a new way of putting Services and APIs in Applications. Please contact the guys working on [this ticket](https://github.com/kyma-project/kyma/issues/13718) for the input.